### PR TITLE
[ADD] runbot_cla

### PR DESCRIPTION
**Pull request by antonylesuisse**
remove useless duplicate status in runbot
load full author and committer name
clean up repo form view
add runbot cla module

### CODE REVIEW

File: runbot/runbot.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ def update_git(self, cr, uid, repo, context=None):
             repo.git(['fetch', '-p', 'origin', '+refs/heads/*:refs/heads/*'])
             repo.git(['fetch', '-p', 'origin', '+refs/pull/*/head:refs/pull/*'])
 
-        fields = ['refname','objectname','committerdate:iso8601','authorname','subject','committername']
+        fields = ['refname','objectname','committerdate:iso8601','author','subject','committer']
```
Comment (antonylesuisse) [inline runbot/runbot.py]: Should i store authorname and authoremail in sperate field ?

File: runbot_cla/runbot.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    _inherit = "runbot.build"
+
+    def job_15_check_cla(self, cr, uid, build, lock_path, log_path):
+        # TODO manage exception in schedule and remove excepts
+        build._log('check_cla', 'Check cla')
+        if build.repo_id.token:
+            cla_glob = build.path("doc/contributors/*.md")
+            cla = ''.join(open(f).read() for f in glob.glob(cla_glob))
+            cla = cla.lower()
+            mo = re.search('[^ <@]+@[^ @>]+', build.author)
```
Comment (KangOl) [inline runbot_cla/runbot.py]: This regex is too fuzzy.
Use `tools.email_split`
(see https://github.com/odoo/odoo/commit/005e24fada851efaed112fce4244a3af225d8f74)

File: runbot_cla/runbot.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            state = "failure"
+            if mo:
+                email = mo.group(0).lower()
+                if re.match('.*(odoo|openerp|tinyerp).com$',email):
+                    state = "success"
+                if cla.find(mo.group(0).lower()) != -1:
+                    state = "success"
+            status = {
+                "state": state,
+                "target_url": "http://github.com/odoo/odoo/",
```
Comment (KangOl) [inline runbot_cla/runbot.py]: use `https://www.odoo.com/cla` (which can redirect anywhere)

File: runbot_cla/runbot.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                "context": "continuous-integration/cla"
+            }
+            build.repo_id.github('/repos/:owner/:repo/statuses/%s' % build.name, status)
+        return -2
+
+    def job_20_test_all(self, cr, uid, build, lock_path, log_path):
+        return -2
+
+    def job_30_run(self, cr, uid, build, lock_path, log_path):
+        return self.spawn(['sleep','3600'], lock_path, log_path, cpu_limit=None)
```
Comment (KangOl) [inline runbot_cla/runbot.py]: 1 hour?

**Pull request merged**